skimage/exposure/tests/test_exposure.py

```diff
@@ -726,10 +732,10 @@ def test_adjust_inv_sigmoid_cutoff_half():
     assert_array_equal(result, expected)
 
 
-def test_negative():
+def test_adjust_sigmoid_negative():
     image = np.arange(-10, 245, 4).reshape((8, 8)).astype(np.double)
     with testing.raises(ValueError):
-        exposure.adjust_gamma(image)
+        exposure.adjust_sigmoid(image)
```

Comment:
```suggestion
        exposure_function(image)
```

Reply:
I introduced this change because I noticed this use of `adjust_gamma` in the `adjust_sigmoid` section.
You are right @grlee77 , the all three functions make use of `exposure._assert_non_negative` function , but only `adjust_gamma` was tested to raise the expected `ValueError`.

Reply:
Thank you @grlee77, I made the requested changes in afdbc5b :wink: